src/otx/algo/classification/backbones/efficientnet.py

```diff
@@ -439,17 +437,14 @@ def __init__(
         bn_eps: float = 1e-5,
         in_channels: int = 3,
         in_size: tuple[int, int] = (224, 224),
-        dropout_cls: dict | None = None,
         pooling_type: str | None = "avg",
         bn_eval: bool = False,
         bn_frozen: bool = False,
         instance_norm_first: bool = False,
-        pretrained: bool = False,
         **kwargs,
     ):
         super().__init__(**kwargs)
         self.num_classes = 1000
-        self.pretrained = pretrained
```

Comment:
Could you remove docstring for this argument also?

Reply:
Thanks - Done: https://github.com/openvinotoolkit/training_extensions/pull/3864/commits/3aafcefc3dfa9ab4f84533c5e4e824ec42a5e819